postgres_lambda_strategy/variables.tf

```diff
@@ -12,3 +12,9 @@ variable "subnet_ids" {
   description = "VPC subnet ids for the function"
   type        = list(string)
 }
+
+variable "tags" {
```

Comment:
Since this example is intended to be a bit more of a dropin and go, including the tags for the AWS stuff makes sense, since most people will have some sort of tagging requirement.